bootstrap/fabric/src/main/java/org/geysermc/geyser/platform/fabric/mixin/client/IntegratedServerMixin.java

```diff
@@ -58,6 +58,7 @@ private void onOpenToLan(GameType gameType, boolean cheatsAllowed, int port, Cal
             // Ensure player locale has been loaded, in case it's different from Java system language
             GeyserLocale.loadGeyserLocale(this.minecraft.options.languageCode);
             // Give indication that Geyser is loaded
+            assert this.minecraft.player != null;
```

Comment:
These do nothing unless a special flag is enabled, and aren't used here anyway.
Objects.requireNonNull is your best bet to get rid of the warning

Reply:
Thanks; fixed in https://github.com/GeyserMC/Geyser/pull/4238/commits/f404b6d8e4a5f60bc074d8f64023674afe6aaa14

Reply:
everywhere else too pls

Reply:
https://github.com/GeyserMC/Geyser/pull/4238/commits/005e0119cce2b5aa68cc7db44409650be0a6cfcb